.github/scripts/lint-ecosystem.js

```diff
@@ -19,15 +19,29 @@ module.exports = async function ({ core }) {
   let inCommunitySection = false
   let modules = []
   let hasImproperFormat = false
+  let moduleName
 
   for await (const line of rl) {
     lineNumber += 1
+
+    if (line.startsWith('- [') === true) {
```

Comment:
We can cache this `startsWith` since it is used again later.

Reply:
Thank you for your suggestion, I've added a commit to address this change.
I've also added two more commits:
- One commit to extend the functionality of the ordering to all sections.
- Another commit to fix out of order modules in the `Core` section.
If you feel like the last two commits are irrelevant to this PR, or not required, I can drop them if you wish